lib/index.js

```diff
@@ -172,7 +172,6 @@ internals.parseServer = function (server, relativeTo) {
 
 internals.parsePlugin = function (plugin, relativeTo) {
 
-    plugin = Hoek.cloneWithShallow(plugin, ['options']);
```

Comment:
I need to have deep options, they can't be swallow.

Wouldn't adding 'plugin.options' to the swallow copy would yield the exact same object?
Since the only swallow property in the copied object is 'options' and then you overwrite it with a deep version.